File: src/main/java/org/adoptopenjdk/jheappo/objects/RootStickyClass.java
package org.adoptopenjdk.jheappo.objects;

/*
 * Copyright (c) 2018 Kirk Pepperdine.
 * Licensed under https://github.com/AdoptOpenJDK/jheappo/blob/master/LICENSE
 * Instructions: https://github.com/AdoptOpenJDK/jheappo/wiki
 */


import org.adoptopenjdk.jheappo.io.HeapProfileRecord;

public class RootStickyClass extends HeapObject {

    public static final int TAG = 0x05;

    public RootStickyClass(HeapProfileRecord buffer) {
        super(buffer);
    }
}

File: src/main/java/org/adoptopenjdk/jheappo/objects/RootStickyClass.kt
package org.adoptopenjdk.jheappo.objects

/*
 * Copyright (c) 2018 Kirk Pepperdine.
 * Licensed under https://github.com/AdoptOpenJDK/jheappo/blob/master/LICENSE
 * Instructions: https://github.com/AdoptOpenJDK/jheappo/wiki
 */

import org.adoptopenjdk.jheappo.heapdump.EncodedChunk

class RootStickyClass(buffer: EncodedChunk) : HeapObject(buffer) {
    companion object {
        const val TAG = 0x05
    }
}
